scripts/modules/models-customer.js

```diff
@@ -387,10 +389,15 @@
                     returnHistory.trigger('returndisplayed', id);
                 });
             });
+            quoteHistory.lastRequest = {
+                pageSize: 5
+            };
 
-            _.defer(function (cust) {
-                cust.getCards();
-            }, self);
+            if (window.location.pathname.toLowerCase().indexOf("selleraccount") < 0) {
```

Comment:
Don't like this fix. It's a hack and I think we need to find a better solution.

You might try checking if the user is a Seller like how `isNonPurchaser` checks the behaviors. Maybe a `isSellerAccount` that checks `isSalesRep`.

Alternatively I would keep `EditableCustomer` for Buyers only and create a `EditableSeller` or `EditableSellerAccount` for use with Sellers only. Long-term that might be the better approach. If you can separate them without tons of work I think I would prefer that.

Reply:
added **if (!isSalesRep)** check